airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -499,3 +500,149 @@ def execute_complete(self, context: Context, event: dict[str, Any] | None = None
+
+    :param aws_conn_id: The Airflow connection used for AWS credentials.
+        If this is ``None`` or empty then the default boto3 behaviour is used. If
+        running Airflow in a distributed manner and aws_conn_id is None or
+        empty, then default boto3 configuration would be used (and must be
+        maintained on each worker node).
+    :param region_name: AWS region_name. If not specified then the default boto3 behaviour is used.
+    :param verify: Whether or not to verify SSL certificates. See:
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html
+    :param botocore_config: Configuration dictionary (key-values) for botocore client. See:
+        https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html
+    """
+
+    aws_hook_class = GlueDataQualityHook
+    template_fields: Sequence[str] = (
+        "datasource",
+        "role",
+        "recommendation_run_kwargs",
+    )
+
+    template_fields_renderers = {"datasource": "json", "recommendation_run_kwargs": "json"}
+
+    ui_color = "#ededed"
+
+    def __init__(
+        self,
+        *,
+        datasource: dict,
+        role: str,
+        number_of_workers: int = 5,
+        timeout: int = 2880,
+        show_results: bool = True,
+        recommendation_run_kwargs: dict[str, Any] | None = None,
+        wait_for_completion: bool = True,
+        waiter_delay: int = 60,
+        waiter_max_attempts: int = 20,
+        deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),
+        aws_conn_id: str | None = "aws_default",
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.datasource = datasource
+        self.role = role
+        self.number_of_workers = number_of_workers
+        self.timeout = timeout
+        self.show_results = show_results
+        self.recommendation_run_kwargs = recommendation_run_kwargs or {}
+        self.wait_for_completion = wait_for_completion
+        self.waiter_delay = waiter_delay
+        self.waiter_max_attempts = waiter_max_attempts
+        self.deferrable = deferrable
+        self.aws_conn_id = aws_conn_id
+
+    def execute(self, context: Context) -> str:
+        glue_table = self.datasource.get("GlueTable", {})
+
+        if not glue_table.get("DatabaseName") or not glue_table.get("TableName"):
+            raise AttributeError("DataSource glue table must have DatabaseName and TableName")
+
+        self.log.info("Submitting AWS Glue data quality recommendation run with %s", self.datasource)
+
+        try:
+            response = self.hook.conn.start_data_quality_rule_recommendation_run(
+                DataSource=self.datasource,
+                Role=self.role,
+                NumberOfWorkers=self.number_of_workers,
+                Timeout=self.timeout,
+                **self.recommendation_run_kwargs,
+            )
+        except ClientError as error:
+            print(error)
```

Comment:
Left over? Or if it is intended, we should log instead

Reply:
Good catch, Sorry my bad i left it there. i have removed now. thank you.